### ISSUE TITLE
How to hide the controls and show only video view?

### ISSUE THREAD

**Issue by hanishcodebrew** [abbasfreestyle/react-native-af-video-player#70]
Hi,

@abbasfreestyle first of all I would like to thank you for this awesome library. I am using it for past many days. You have handled almost everything. My question is how to hide the time seek, play button. I only want to show the video. 

Thanks

**Comment by vforvasile** [comment]
Hey @abbasfreestyle , this library was built specifically for the controls, UI, fullscreen mod and so on. If you need a play/pause sample without additional UI elements, head to **react-native-video** and use the basic example.

Another option would be to set theme color to transparent on the item you don't want to see (  title: "transparent", more: "transparent", center: "transparent" )  - this will make the Play/Pause icon, More and Title transparent. (**highly unrecommended** for the bottom icons though).

Good luck!

**Comment by prevuelta** [comment]
I would mention though if you want to use both modules, react-native throws an error `Tried to register two views with the same name RTCVideo`
so an option would be great for when you need videos both with and without controls. 
Or alternatively fix the conflict.

**Comment by hanishcodebrew** [comment]
Yes, it would be great, if both the options are provided as mentioned by @prevuelta . For the time being I am using the approach mentioned by @vforvasile . But still seekbar is not getting transparent.
